docs/install-1.7.md

```diff
@@ -18,8 +18,6 @@ authentication and authorization.
 
 The Service Catalog utilizes API aggregation to present its API.
 
-# Prerequisites
```

Comment:
We should either keep this one and get rid of the next line, or adjust all the step numbering. Right now this doc has "Step 1" twice.

Arguably, given the size of the prereqs,we should do the renumbering.

Reply:
I agree with doing the renumbering.

Reply:
I am OK to do the renumbering